### PR TITLE
DocBlockHelper Regression Test

### PR DESCRIPTION
**Pull request by ravage84**
As asked by @dereuromark at https://github.com/cakephp/bake/pull/514#issuecomment-477235431

Sorry for sneaking in some unrelated improvements.
Hope they are fine, too.

Didn't have time to cover all methods, though. :sleeping_bed:

### CODE REVIEW

**Comment by dereuromark** [comment]
I better start running IdeHelper tests against bake dev-master :P
Early warning system

File: src/View/Helper/DocBlockHelper.php [hunk 1]
```diff
@@ -13,7 +13,10 @@
  */
 class DocBlockHelper extends Helper
 {
-    protected $_annotationSpacing = true;
+    /**
+     * @var bool Whether to add a blank line between different class annotations
+     */
+    protected $_classAnnotationSpacing = true;
```
Comment (dereuromark) [inline src/View/Helper/DocBlockHelper.php]: changing this should be avoided if we want to make this a patch release.

**Comment by dereuromark** [comment]
Looks good otherwise. And IdeHelper is also happy.

File: tests/TestCase/View/Helper/DocBlockHelperTest.php [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+    /**
+     * Tests the propertyHints method
+     *
+     * @return void
+     * @covers ::propertyHints
+     */
+    public function testPropertyHints()
+    {
+        $this->markTestIncomplete('Not implemented yet');
```
Comment (markstory) [inline tests/TestCase/View/Helper/DocBlockHelperTest.php]: Can we implement these or delete the tests?

Comment (ravage84) [reply tests/TestCase/View/Helper/DocBlockHelperTest.php]: Sure we can, either. I just had no time yet. I just wanted to increase the visibility for the lack of coverage in the mean time.

File: src/View/Helper/DocBlockHelper.php (hunk 1 above)
Comment (ravage84) [reply src/View/Helper/DocBlockHelper.php]: I would be fine to make it a minor release, too.

Refs: https://book.cakephp.org/3.0/en/contributing/backwards-compatibility.html#working-on-cakephp

Comment (ADmad) [reply src/View/Helper/DocBlockHelper.php]: Is there any actual feature change which warrants a minor release? I haven't been following the issue but this seems to be related to just changing the whitespacing in generated code.

Comment (dereuromark) [reply src/View/Helper/DocBlockHelper.php]: I would also not change the name here, the added doc block info suffices IMO.

**Pull request merged**